batch/batch/driver/instance_collection_manager.py

```diff
@@ -16,10 +17,11 @@ def __init__(self, app, machine_name_prefix):
         self.db: Database = app['db']
         self.zone_monitor: ZoneMonitor = app['zone_monitor']
         self.machine_name_prefix = machine_name_prefix
-        self.inst_coll_regex = re.compile(f'{self.machine_name_prefix}(?P<inst_coll>[^-]+)-.*')
+        self.inst_coll_regex = re.compile(f'{self.machine_name_prefix}(?P<inst_coll>.*)-.*')
```

Comment:
inst_coll can't be empty, right? Should this be .+?

Reply:
It used to be empty before when we did the pools migration. I'll have to double check with where this is used to make sure we can switch the regex.

Reply:
Let's hold off on this for now. I'm just worried about the case where in the future we want to do something with instances that date pre-pool change.